mq/publishers.go

```diff
@@ -108,6 +111,163 @@ func PublishSingleHostPeerUpdate(ctx context.Context, host *models.Host, deleted
 	return publish(host, fmt.Sprintf("peers/host/%s/%s", host.ID.String(), servercfg.GetServer()), data)
 }
 
+// FlushNetworkPeersToHost - sends all the peers in the network to the host.
+func FlushNetworkPeersToHost(host *models.Host, hNode *models.Node, networkNodes []models.Node) error {
```

Comment:
This should eventually be changed to PublishPeerUpdate or PublishNetworkPeerUpdate, understand that it can't be done yet

Reply:
this won't be changed to PublishPeerUpdate...
PublishPeerUpdate is consolidated host update that is called  minimally going forward only when required